pkg/cli/cmd/credential/show/objectformats.go

```diff
@@ -87,7 +87,26 @@ func credentialFormatAWS() output.FormatterOptions {
 			},
 			{
 				Heading:  "ACCESSKEYID",
-				JSONPath: "{ .AWSCredentials.AccessKeyID }",
+				JSONPath: "{ .AWSCredentials.AccessKey.AccessKeyID }",
+			},
+		},
+	}
+}
+
+func credentialFormatAWSIRSA() output.FormatterOptions {
+	return output.FormatterOptions{
+		Columns: []output.Column{
+			{
+				Heading:  "NAME",
+				JSONPath: "{ .Name }",
+			},
+			{
+				Heading:  "REGISTERED",
+				JSONPath: "{ .Enabled }",
+			},
+			{
+				Heading:  "ROLEARN",
+				JSONPath: "{ .AWSCredentials.IRSA.RoleARN }",
```

Comment:
Do we also want to add `HEADING: Kind` ?

Reply:
I was thinking its a bit evident since the column name changes between rolearn and accesskey. but looks like azure credentials have a kind column. I will implement this. cc @Reshrahim @willtsai

Reply:
implemented